tests/libres_tests/res/enkf/test_enkf_load_results_manually2.py

```diff
@@ -0,0 +1,25 @@
+import logging
+import pytest
+
+from ecl.util.util import BoolVector
+from res.enkf import EnKFMain
+
+
+@pytest.mark.parametrize("lazy_load", [True, False])
+def test_load_results_manually(setup_case, caplog, monkeypatch, lazy_load):
+    """
+    This little test does not depend on Equinor-data and only verifies
+    the lazy_load flag in forward_load_context plus memory-logging
+    """
+    if lazy_load:
+        monkeypatch.setenv("ERT_LAZY_LOAD_SUMMARYDATA", str(lazy_load))
+    res_config = setup_case("local/snake_oil", "snake_oil.ert")
+    ert = EnKFMain(res_config)
+    load_from = ert.getEnkfFsManager().getFileSystem("default_0")
+    ert.getEnkfFsManager().switchFileSystem(load_from)
+    realisations = BoolVector(default_value=False, initial_size=25)
+    realisations[0] = True  # only need one to test what we want
+    with caplog.at_level(logging.INFO):
+        loaded = ert.loadFromForwardModel(realisations, 0, load_from)
```

Comment:
If you really want this test to do something I guess you will need to populate the runpath directories with summary cases - e.g. created with `EclSum::createWriter()`

Reply:
Same as previous comment?